GitUI/BranchTreePanel/RepoObjectsTree.Nodes.Submodules.cs

```diff
@@ -93,6 +93,12 @@ protected override string NodeName()
 
             public void Open()
             {
+                if (!Directory.Exists(Info.Path))
+                {
+                    MessageBoxes.SubmoduleDirectoryDoesNotExist(null, Info.Path, Info.Text);
```

Comment:
Give the null parameter name. Why is it null? The error will be unparented, and can get lost behind other windows.

Reply:
changed to `this` where possible (not in side panel or the generic Launch).